cdap-app-fabric/src/main/java/io/cdap/cdap/internal/operation/OperationLifecycleManager.java

```diff
@@ -78,6 +80,27 @@ public boolean scanOperations(ScanOperationRunsRequest request, int txBatchSize,
     return currentLimit == 0;
   }
 
+   /**
+ * Retrieves details of an operation run identified by the provided {@code OperationRunId}.
+ *
+ * @param runId The unique identifier for the operation run.
+ * @return An {@code OperationRunDetail} object containing information about the specified operation run.
+ * @throws OperationRunNotFoundException If the specified operation run is not found.
+ */
+  public OperationRunDetail getOperationRun(OperationRunId runId)
+      throws IOException, OperationRunNotFoundException {
+    OperationRunDetail operationRunDetail =
```

Comment:
inline variable i.e return from here directly no need to create a valibale

Reply:
done